src/rules/selector-descendant-combinator-no-non-space/README.md

```diff
@@ -0,0 +1,32 @@
+# selector-descendant-combinator-no-non-space
+
+Disallow non-space characters for descendant combinators of selectors.
+
+```css
+   .foo .bar .baz { color: pink; }
+/**    ↑         ↑
+ * These are descendant combinators */
+```
+
+This rule ensures that there is only single space are used and verifies that tabs, newlines, and  multiple spaces are not used for descendant combinators of selectors.
```

Comment:
`This rule ensures that only a single space is used and ensures no tabs, newlines, nor multiple spaces are used for descendant combinators of selectors.`

That's my take on more succinct verbiage?